src/hotspot/share/runtime/globals.hpp

```diff
@@ -763,7 +763,7 @@ const size_t minimumSymbolTableSize = 1024;
   notproduct(bool, PrintInlineLayout, false,                                \
           "Print field layout for each inline type")                        \
                                                                             \
-  notproduct(bool, PrintInlineArrayLayout, false,                           \
+  notproduct(bool, PrintFlatArrayLayout, false,                             \
           "Print array layout for each inline type array")                  \
                                                                             \
   product(intx, InlineArrayElemMaxFlatSize, -1,                             \
```

Comment:
InlineArrayElementMaxFlatSize -> FlatArrayElementMaxSize ?

Reply:
How about if I change InlineArrayElementMaxFlatSize and InlineArrayElemMaxFlatOops in the next change because changing them affects a lot of tests and would make this change even bigger?